mmv1/products/compute/api.yaml

```diff
@@ -13545,6 +13550,56 @@ objects:
               - :ERRORS_ONLY
               - :TRANSLATIONS_ONLY
               - :ALL
+      - !ruby/object:Api::Type::Array
+        name: rules
```

Comment:
Are these fields updatable? Can we also add an update test for this feature?

Reply:
added